File: Programming Basics/For Loop - Exercises/02.HalfSumElement.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int numbers = int.Parse(Console.ReadLine());
        int sum = 0;
        int max = 0;

        for (int i = 0; i < numbers; i++)
        {
            int number = int.Parse(Console.ReadLine());
            sum += number;

            if (number > max)
                max = number;     
        }
        if ((sum - max) == max)
            Console.WriteLine("Yes\r\n" +
                $"Sum = {sum - max}");
        else
            Console.WriteLine("No\r\n" +
                $"Diff = {Math.Abs(max - (sum - max))}");
    }
}

File: Programming Basics - Python/For Loop - Exercises/02.HalfSumElement.py
number = int(input())

biggest_number = 0
total_sum = 0

for num in range(0, number):
    n = int(input())

    if biggest_number < n:
        biggest_number = n

    total_sum += n

if biggest_number == total_sum - biggest_number:
    print("Yes")
    print(f"Sum = {biggest_number}")

else:
    print("No")
    print(f"Diff = {abs(biggest_number - (total_sum - biggest_number))}")